torch/utils/cpp_extension.py

```diff
@@ -1017,6 +1029,16 @@ def CUDAExtension(name, sources, *args, **kwargs):
     constructor. Full list arguments can be found at
     https://setuptools.pypa.io/en/latest/userguide/ext_modules.html#extension-api-reference
 
+    Note!
+    If passing in the ``py_limited_api`` flag signifying that the extension uses
+    only the Python limited API, CUDAExtension will respect the flag and SKIP
+    linking torch_python with your custom extension!!!! This means that your
+    extension CANNOT rely on any APIs provided in torch_python!!!! Notably, your
+    extension cannot call PYBIND11_MODULE, which is commonly used to create a
+    Python module associated with the extension. The APIs in torch_python are
+    blatantly NOT within the Python limited API, which is why we avoid linking
+    torch_python when the ``py_limited_api`` is specified.
```

Comment:
I think we could make this shorter and to the point? I feel like there are only 2 ideas here:
- PyTorch python API (as provided in libtorch_python) cannot be built with py_limited_api
- When this flag is passed, it is the user responsibility in their library to not use APIs from libtorch_python (in particular pytorch/python bindings) and only use APIs from libtorch (aten objects, operators and registering operations through the dispatcher for access from python).

Reply:
oki let me try again!